doc/related_projects.rst

```diff
@@ -253,17 +208,9 @@ Other packages useful for data analysis and machine learning.
 - `Seaborn <https://stanford.edu/~mwaskom/software/seaborn/>`_ Visualization library based on
   matplotlib. It provides a high-level interface for drawing attractive statistical graphics.
 
-- `Deep Learning <http://deeplearning.net/software_links/>`_ A curated list of deep learning
-  software libraries.
-
 Recommendation Engine packages
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
- - `GraphLab
-   <https://turi.com/products/create/docs/graphlab.toolkits.recommender.html>`_
```

Comment:
Very unclear how to install this, of what is the corresponding Github repo. I think it's https://github.com/jegonzal/PowerGraph no updates since 2016.